packages/libisitweeka/src/index.ts

```diff
@@ -143,43 +145,222 @@ export default class IsItWeekA {
 
 		const ics = await baseResponse.text();
 
-		// console.log(ics);
+		// New Version!
```

Comment:
Do you think we could add a doc comment to the start of this function to explain the algo, or essentially add to a comment if one already exists about what the new bit does (looks like it expands out all the recurrences then pipes the 'expanded' calendar into the classic algorithm?

Also consider moving new part to a `expandRecurrences` function?

Reply:
Old comments had gotten a little lost, and a few new ones have been added to maintain clarity